meta-security-smack/lib/oeqa/runtime/smack.py

```diff
@@ -5,6 +5,12 @@
 from oeqa.oetest import oeRuntimeTest, skipModule
 from oeqa.utils.decorators import *
 
+def get_files_dir():
```

Comment:
oeqa.utils.helper is not yet in OE-core, so we cannot depend on that in this layer. Like deploy_files.bbclass itself we need to duplicate it for now.